frontend/src/app/components/dashboard/db-table-view/db-table-view.component.html

```
@@ -1,31 +1,56 @@
+<div class="pull-refresh-indicator"
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)